diagnostic.md

```diff
@@ -14,16 +14,20 @@ list items (`li`) using jQuery? This is intentionally vague. (Assume starting
 at the `OL` DOM node).
 
 <!-- your answer starts here -->
+ol li {
+
+}
+body li {
 
+}
```

Comment:
Make sure you have the proper jQuery formatting for questions 1 and 2: `$('ol').children();`